test/nuts/testMatrix.ts

```diff
@@ -15,10 +15,11 @@ const env = new Env();
  * The executables we want to test against. These can be toggled with environment variables
  */
 export const EXECUTABLES = [
-  {
-    path: 'sfdx',
-    skip: !env.getBoolean('PLUGIN_SOURCE_TEST_SFDX', false),
-  },
+  // Commenting this as we no longer have identically named source-plugin and toolbelt commands
```

Comment:
why keep?

Reply:
Just in case it can be used with sf.  But we can always get it from git history so I'll remove it.